src/Panel/File.php

```diff
@@ -64,6 +64,19 @@ public function breadcrumb(): array
 		return $breadcrumb;
 	}
 
+	/**
+	 * Returns array of names for which
+	 * header buttons should be displayed
+	 */
+	public function buttons(): array
+	{
+		return $this->model->blueprint()->buttons() ?? [
+			"preview",
+			"settings",
+			"languages"
+		];
```

Comment:
@bastianallgeier or do you think the defaults should rather go into` PageBlueprint`/`FileBlueprint`... ?